### DISCUSSION TITLE
How to join the slack Channel?

### DISCUSSION THREAD

**Discussion by dimanzt** [sustainable-computing-io/kepler#979] [Q&A] [answered]
Dear all,
Hi,
I'm new with Kepler and have started exploring it.
Is there a way that I could join the slack channel? email: diman.zt@gmail.com

**Comment by rootfs** [comment, marked as answer]
Please join the kepler-project slack channel at CNCF slack
https://communityinviter.com/apps/cloud-native/cncf
